apps/web/src/pages/templates/components/TriggerSnippetTabs.tsx

```diff
@@ -19,14 +20,20 @@ export function TriggerSnippetTabs({ trigger }: { trigger: INotificationTrigger
   const toValue = getSubscriberValue(subscriberVariables, (variable) => variable.value || '<REPLACE_WITH_DATA>');
   const payloadValue = getPayloadValue(trigger.variables);
 
+  const reservedValue = triggerSnippetVariables.reduce((acc, variable) => {
```

Comment:
This will not create a lot of rerenders?

Reply:
The `Tabs` component is not memoized, so it will re-render it and all the children when `TriggerSnippetTabs` will render, like it is right now. 😉

The thing that we can improve is memoize the calculations for this variable, so React won't need to do it every time on a new re-render. Actually same for tabs.